packages/grid/_modules_/grid/locales/bgBG.ts

```diff
@@ -0,0 +1,84 @@
+export const bgBG = {
+  components: {
+    MuiDataGrid: {
+      defaultProps: {
```

Comment:
A note for once we work on #984. We will need to adapt the structure. I guess we can create a factory utility to abstract it.

Reply:
I guess that is to be able to support both v4 and v5 at the same time correct?

I'll check what the v4 structure is.